browser/ui/views/location_bar/brave_location_bar_view.cc

```diff
@@ -15,9 +15,6 @@
 #include "components/version_info/channel.h"
 
 void BraveLocationBarView::Init() {
-  // LocationBarView (original) GetTint is called from ctor,
-  // which will not use our overriden function, so call it again here.
-  tint_ = GetTint();
```

Comment:
is this correct? should it be calling the new `CalculateTint`?

Reply:
curious because I did notice the omnibar has a very bright orange surround now (versus a more muted orange)

Reply:
My understanding is that tint is calculated on the fly with this new method, and `tint_` member variable has been removed since there is no need to cache tint any more.

Reply:
but based on the comment I'm not sure if this will continue to function correctly. Can you check git blame to see who wrote this code?